joomla/joomla-cms
# [mod_articles_category] Showon

Pull Request for Issue # .

### Summary of Changes

This adds `showon` attribute to form fields in Articles - Category module.

### Testing Instructions

Code review. Or create Articles - Category module and try different options. Check that fields are shown/hidden correctly.

### Documentation Changes Required
Option from Dynamic Mode Options moved to Module tab. Update help page? https://docs.joomla.org/Help38:Extensions_Module_Manager_Articles_Category
